deps/jemalloc/src/jemalloc.c

```diff
@@ -4483,3 +4492,102 @@ get_defrag_hint(void* ptr) {
 	assert(ptr != NULL);
 	return iget_defrag_hint(TSDN_NULL, ptr);
 }
+
+JEMALLOC_EXPORT JEMALLOC_ALLOCATOR JEMALLOC_RESTRICT_RETURN
+void JEMALLOC_NOTHROW *
+JEMALLOC_ATTR(malloc) JEMALLOC_ALLOC_SIZE(1)
+malloc_with_usize(size_t size, size_t *usize) {
```

Comment:
these 4 functions are copied from je_malloc, etc., only adding the `usize` parameter.